app/src/main/java/org/sopt/dosopttemplate/ui/base/BindingFragment.kt

```diff
@@ -0,0 +1,35 @@
+package org.sopt.dosopttemplate.ui.base
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.annotation.LayoutRes
+import androidx.databinding.DataBindingUtil
+import androidx.databinding.ViewDataBinding
+import androidx.fragment.app.Fragment
+
+abstract class BindingFragment<T : ViewDataBinding>(
+    @LayoutRes private val layoutRes: Int,
+) : Fragment() {
+    private var _binding: T? = null
+    protected val binding get() = requireNotNull(_binding) {
+
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?,
+    ): View? {
+        _binding = DataBindingUtil.inflate(inflater, layoutRes, container, false)
+        binding.lifecycleOwner = viewLifecycleOwner
+        return binding.root
+    }
+
+
+    override fun onDestroyView() {
+        super.onDestroyView()
+        _binding = null
```

Comment:
```suggestion
        _binding = null
        super.onDestroyView()
```

요거 순서!! 피피피도.. 바꿔야겠도라 1차 세미나 하다가 발견함